content/applications/hr/payroll.rst

```diff
@@ -91,32 +87,39 @@ form:
    entry remains 5.5 hours. If :guilabel:`Rounding` is set to :guilabel:`Half Day`, the entry is
    changed to 4 hours. If it is set to :guilabel:`Day`, it is changed to 8 hours.
 
-Working times
--------------
+- :guilabel:`Unpaid in Structures Types`: If the work entry is for unpaid work, specify which pay
+  structure the work entry applies to from the drop-down menu.
+- Check boxes: If any of the items in the list applies to the work entry, check off the box by
+  clicking it. If :guilabel:`Time Off` is checked off, a :guilabel:`Time Off Type` field appears.
+  This field has a drop-down menu to select the specific type of time off, or a new type of time off
+  can be entered.
```

Comment:
There is another check box, the "Keep Time Off Right" field. Can you elaborate on it? (The tool tip does not really help much, at least it did not help me). After some back and forth with translating it seems like this might be closer to "entitlement" instead of "right".

Reply:
I am checking with Quentin on this- I will have this updated when I hear back.

Reply:
OK, this is for Belgium only and affects if time off can be rolled over into the next year, and if it affects how much time they accrue, etc.